ksqldb-functional-tests/src/test/resources/query-validation-tests/multi-joins.json

```diff
@@ -813,6 +813,70 @@
         ]
       }
     },
+    {
+      "name": "stream-stream-stream - outer-outer",
+      "statements": [
+        "CREATE STREAM S1 (ID INT KEY, V0 bigint) WITH (kafka_topic='left', value_format='JSON');",
+        "CREATE STREAM S2 (ID INT KEY, V0 bigint) WITH (kafka_topic='right', value_format='JSON');",
+        "CREATE STREAM S3 (ID INT KEY, V0 bigint) WITH (kafka_topic='right2', value_format='JSON');",
+        "CREATE STREAM OUTPUT as SELECT ROWKEY, s1.V0, s2.V0, s3.V0 FROM S1 FULL OUTER JOIN S2 WITHIN 5 seconds ON S1.ID = S2.ID FULL OUTER JOIN S3 WITHIN 10 seconds ON S1.ID = S3.ID;"
```

Comment:
Names should be all lower case, eg, `s1.v0` and `s1.id = s1.id` an `s1 FULL OUTER JOIN s2`

Reply:
See above -- copy-and-paste from basically all the tests in this file. But I've updated the new ones to your liking.